cudaaligner/src/aligner_global_myers_banded.cpp

```diff
@@ -84,7 +89,8 @@ memory_distribution split_available_memory(const int64_t max_device_memory, cons
     r.query_patterns_memory = static_cast<int64_t>(fmax_device_memory / mem_req_total_per_bp * mem_req_query_patterns);
     r.pmvs_matrix_memory    = static_cast<int64_t>(fmax_device_memory / mem_req_total_per_bp * mem_req_pmvs_matrix);
     r.score_matrix_memory   = static_cast<int64_t>(fmax_device_memory / mem_req_total_per_bp * mem_req_score_matrix);
-    r.remainder             = max_device_memory - (2 * r.sequence_memory + r.results_memory + r.query_patterns_memory + 2 * r.pmvs_matrix_memory + r.score_matrix_memory);
+    r.remainder             = 0;
```

Comment:
Why set to `0` and then set to actual value?

Reply:
Because `get_total_memory_required(r)` depends on `r.remainder`. Probably this needs a comment.
First, the total memory required by everything without the remainder is computed and then the remainder is set to `max_device_memory` minus this required memory.

Reply:
Ahh, yes, now I see it, but it is a bit confusing

Reply:
Yes, a comment would be nice

Reply:
Done.